doc/source/math/geocoordinates.rst

```diff
@@ -107,40 +107,40 @@ Structure
 
     The ship-raw 3D position above or below the surface of the body, relative to the current ship's Center of mass.  You pass in an altitude number for the altitude above "sea" level of the desired location.
 
-Examples Usage
---------------
+Example Usage
+-------------
 
 ::
 
-    SET spot TO LATLNG(10, 20).     // Initialize point at latitude 10,
-                                    // longitude 20
+    // Initialize point at latitude 10, longitude 20:
+    SET spot TO LATLNG(10, 20).
                                     
-    PRINT spot:LAT.                 // Print 10
-    PRINT spot:LNG.                 // Print 20
+    PRINT spot:LAT.     // Print 10
+    PRINT spot:LNG.     // Print 20
     
-    PRINT spot:DISTANCE.            // Print distance from vessel to x
-                                    // (same altitude is presumed)
-    PRINT spot:HEADING.             // Print the heading to the point
-    PRINT spot:BEARING.             // Print the heading to the point
-                                    // relative to vessel heading
+    // Print distance from vessel to x (same altitude is presumed):
+    PRINT spot:DISTANCE.
+    // Print the heading to the point
+    PRINT spot:HEADING.
+    // Print the heading to the point relative to vessel heading:
+    PRINT spot:BEARING.
                                     
-    SET spot TO SHIP:GEOPOSITION.   // Make spot into a location on the
-                                    // surface directly underneath the
-                                    // current ship
+    // Make spot into a location on the surface directly underneath the current ship:
+    SET spot TO SHIP:GEOPOSITION.
                                     
-    SET spot TO LATLNG(spot:LAT,spot:LNG+5). // Make spot into a new
-                                             // location 5  degrees east
-                                             // of the old one
+    // Make spot into a new location 5 degrees east of the old one:
+    SET spot TO LATLNG(spot:LAT,spot:LNG+5).
```

Comment:
I've always understood that full-line comments are preferable to to inline comments, and especially so with line wrapping because they get broken into so many little chunks and take up a lot of vertical space.

If there's a convention here that I'm bucking, it was unintentional and I'll revert it.